client/components/payment-method-selector/index.js

```diff
@@ -0,0 +1,55 @@
+import React, { PropTypes } from 'react';
```

Comment:
Since this is a component that will only be used in the Payment Selection screen, it's better to put it in `client/shared-settings/views` and leave `client/components` only for generic, reusable components (to reduce the clutter). Same for the `PaymentMethod` component.